README.md

```diff
@@ -14,6 +14,8 @@ Documentation
 
 [You can view a list of our capabilities and how to use the API provided here.](docs/README.md)
 
+[Here you can view a list of our limitations.](docs/limitations.md)
+
```

Comment:
Can we also get a link to the docs/architectural-decisions.md if it makes sense?